tests/test_runner.py

```diff
@@ -83,3 +86,39 @@ def test_ansible_test_runner_run(mocker, temp_config_file):  # noqa F811
         ansible_test_runner().run()
         assert not runners.LocalAnsibleTestRunner.called
         assert runners.OpenshiftJobTestRunner.called
+
+
+def test_local_run(mocker, caplog):  # noqa F811
+    mocker.patch.object(subprocess, 'Popen')
+    subprocess.Popen.return_value.stdout = ['stdout_result']
+    subprocess.Popen.return_value.wait.return_value = 0
+    caplog.set_level(logging.INFO)
+
+    metadata = SimpleNamespace(
+        namespace='test_ns', name='test_name', version='test_version')
+    runner = runners.local.LocalAnsibleTestRunner(metadata=metadata)
+    runner.run()
+
+    assert len(caplog.records) == 4
```

Comment:
I'm not sure what all loggers caplog ends up capturing by default. But if it's root logger or similar, it may be kind of fragile to check the total number of log records since other modules could log things that don't effect normal behavior.